nomad/utils/text.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+import re
+import unicodedata
+
+
+_punct_re = re.compile(r'[\t !"#$%&\'()*\-/<=>?@\[\\\]^_`{|},.]+')
+
+
+def slugify(text):
+    """ Returns a ASCII-only slug version of the considered text. """
+    result = []
+    for word in _punct_re.split(text.lower()):
+        word = unicodedata.normalize('NFKD', word).encode('ascii', 'ignore').decode('ascii')
```

Comment:
Oh, yes, I see, you mean https://github.com/lxd-nomad/lxd-nomad/blob/improve_multi_containers_names/nomad/cli/project.py#L20

Yes, let's enforce a project name. Using the folder name is going to give us problems. What are we going to do when chinese people run nomad in a folder with 100% chinese characters?